doc/changes.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+
+## Release v0.8.0 (2022/12/28)
+
+[![Digital Object Identifier for PyGMT v0.8.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7481934.svg)](https://doi.org/10.5281/zenodo.7481934)
+
+### Highlights
+
+* 🎉 **Eighth minor release of PyGMT** 🎉
+* Wrapped ternary
+*
```

Comment:
```suggestion
* Add intro tutorial section for creating contour maps
* Easy access for various remote datasets
```
maybe the formulation needs to be polished a litte bit 😉

Reply:
I like these suggestions! Feel free to polishe me updated formulations a second time.
But if we state the intro to contour maps as highlight, I feel we should make sure that the similar "normal" tutorial for contour maps is correctly displayed (see PR#2260).